<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_5">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Choose Map Config File: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="openmapconfigbutton">
           <property name="text">
            <string>Open...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="mapload_label">
           <property name="text">
            <string>Map Config File: None</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Choose Robot Config File: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="openrobotconfigbutton">
           <property name="text">
            <string>Open...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="robotload_label">
           <property name="text">
            <string>Robot Config File: None</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="loadconfigbutton">
         <property name="text">
          <string>Load Configs</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Click with left mouse button to append new waypoint.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Or add a pre-defined path in form &lt;(x1,y1),(x2,y2),...&gt;:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="pathinput"/>
           </item>
           <item>
            <widget class="QPushButton" name="loadpathbutton">
             <property name="text">
              <string>Load Path</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Switch between:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="localcoordbutton">
               <property name="text">
                <string>Local Coordinates</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="globalcoordbutton">
               <property name="text">
                <string>Global Coordinates</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Number of weight combinations:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="scale_box">
               <item>
                <property name="text">
                 <string>1000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>729</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>512</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>343</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>216</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>125</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_8">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="deletepathbutton">
             <property name="text">
              <string>Delete Full Path</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deletepointbutton">
             <property name="text">
              <string>Delete Last Point</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Please create a project name as a reference:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="outputfolder">
             <property name="text">
              <string>project_folder_name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="calculatebutton">
             <property name="text">
              <string>Calculate Paths</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="erroroutput">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QWidget" name="mapwidget" native="true">
           <property name="minimumSize">
            <size>
             <width>700</width>
             <height>400</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Paths are being calculated.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="successmsg">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
